.github/workflows/build.yml

```diff
@@ -23,8 +23,9 @@ jobs:
         uses: cpp-linter/cpp-linter-action@v2
         id: linter
         with:
+          version: 14
           style: file
-          tidy-checks: ""
+          tidy-checks: "*"
```

Comment:
Have you tried to omit the parameter completely? According to the doc the default seems fine to me:

https://github.com/cpp-linter/cpp-linter-action#tidy-checks

Reply:
Yes, I tried it before and now in the latest run (with the version 14) again. But still [no source files need checking](https://github.com/engelmi/orch/actions/runs/3646509741/jobs/6157668920#step:4:113)

Reply:
Doesn't that mean that we comply to all checks?

Reply:
Also by default `files-changed-only` is set to `true` and we don't change any source code files here, so if you want to check the whole repo then you probably need to set it to `false`

Reply:
I think so, yes. And then I'd assume the linter to sound the alarm due to the magic numbers in the test directory (which are fine in that case) - but it doesn't. 

Just saw that the [style](https://github.com/cpp-linter/cpp-linter-action/blob/v2.2.4/action.yml#L16-L21) argument only applies to the `.clang-format` file, not `.clang-tidy` containing the linting rules we want. I don't want to duplicate the rules for local and CI usage.